runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -26,9 +26,8 @@
  */
 package net.runelite.client.plugins.xptracker;
 
-import java.awt.BorderLayout;
-import java.awt.Color;
-import java.awt.Dimension;
+import java.awt.*;
```

Comment:
Looks like IntelliJ decided to import all here. I think I need to change my IDE settings, will follow the `Wiki` guide on it shortly.